moduleroot/Rakefile

```diff
@@ -7,3 +7,27 @@ PuppetLint.configuration.fail_on_warnings = true
 PuppetLint.configuration.send('<%= check %>')
 <% end -%>
 PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "pkg/**/*.pp"]
+
+desc 'Generate pooler nodesets'
+task :gen_nodeset do
+  require 'beaker-hostgenerator'
+  require 'securerandom'
+  require 'fileutils'
+
+  agent_target = ENV['TEST_TARGET']
```

Comment:
Good idea. I couldn't help but think that if we're setting a default we should warn the user what we're doing. I've got an update incoming that does both.